### ISSUE TITLE
[MNG-1642] Incorrect APT markup applied to the Title of the guide-releasing.apt 

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#3412]
**[trosenbaum](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=trosenbaum)** opened **[MNG-1642](https://issues.apache.org/jira/browse/MNG-1642?redirect=false)** and commented

The incorrect APT markup has been used on the title of the following mini guide resource:

http://svn.apache.org/viewcvs.cgi/maven/site/trunk/src/site/apt/guides/mini/guide-releasing.apt

The document uses the "----" instead of  " ----", (the second example includes a space).  This is instructing the maven-site plugin to render a box around the content.  This issue is major because it does not present the information to the user in a clear manner.

---

**Affects:** 2.0.1

**Attachments:**
- [MNG-1642-maven-site.patch](https://issues.apache.org/jira/secure/attachment/12711534/MNG-1642-maven-site.patch) (_347 bytes_)
- [screenshot-1 - Showing the rendered error.jpg](https://issues.apache.org/jira/secure/attachment/12711600/screenshot-1+-+Showing+the+rendered+error.jpg) (_201.86 kB_)
- [screenshot-2 - Showing the rendering after patch has been applied.jpg](https://issues.apache.org/jira/secure/attachment/12711535/screenshot-2+-+Showing+the+rendering+after+patch+has+been+applied.jpg) (_231.38 kB_)

**Comment by jira-importer** [comment]
**[trosenbaum](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=trosenbaum)** commented

The above patch corrects the APT format used to describe the title for the document.  All this patch does is place a space before the first dashed line, (that preceeds the author names).  This stops the maven-site plugin from including the content inside a box, (normally used to show examples and code).  This will allow the guide to be correctly rendered for other to read.

Test -
I have checked out all of the maven-site content, (from the trunk) and run the plugin over the project.  This recreated the issue in live.  I then added the space and rerun the plugin and the mini guide for the use of the maven-release plugin was correctly rendered.
